.gitlab-ci.yml

```diff
@@ -990,7 +990,16 @@ workflow:
     when: never
 
 .on_system_probe_or_e2e_changes_or_manual:
+  - <<: *if_disable_e2e
```

Comment:
<!-- Warning highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes and they are blockers. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question. -->
**🔨 warning**
Makes sense to disable these to, this rule is used by KMT, which are hybrid e2e tests - makes even more sense to skip them on integrations core PRs, but maybe we want a dedicated label cc @usamasaqib

Reply:
Yep indeed these ones are not "classic" e2e test. I will revert the modification for this

Reply:
`RUN_E2E_TESTS` is false on only build pipelines, which makes me think we should change the label to `ONLY_BUILD` or something similar. It is off when triggered as a child pipeline from `integrations-core` on dependencies changes. I think it is safe to skip system probe in that case

Reply:
With the current changes the difference with before the change is:
- This job will run on release branches and on RC pipelines, while it was running only on main
- It will run if `RUN_E2E_TESTS==auto`
- It never run if `RUN_E2E_TESTS==off` 
As Paola said, for now `RUN_E2E_TESTS` is only set to off for pipeline triggered from `integrations-core`
If you'd prefer to have the execution of these tests less coupled with the execution of the other "classic" new-e2e tests ( which is understandable since these one are launched before the package build, while all the other need the datadog-agent package to be built) I can remove these changes!

Reply:
I would prefer to keep these un-coupled from new-e2e for now.

Reply:
It is still uncoupled, the variable name is confusing and we will change it soon, merging it with `RUN_KITCHEN_TESTS`. Do you want system-probe owned tests to run on pipelines triggered by integrations-core ?